WORKSPACE

```diff
@@ -12,14 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
 git_repository(
     name = "build_bazel_rules_apple",
     remote = "https://github.com/bazelbuild/rules_apple.git",
-    commit = "7ea0557",
+    tag = "0.9.0",
+)
+
+load(
+    "@build_bazel_rules_apple//apple:repositories.bzl",
+    "apple_rules_dependencies",
+)
+
+apple_rules_dependencies()
+
+git_repository(
+    name = "build_bazel_rules_swift",
+    remote = "https://github.com/bazelbuild/rules_swift.git",
+    tag = "0.4.0",
+)
+
+load(
+    "@build_bazel_rules_swift//swift:repositories.bzl",
+    "swift_rules_dependencies",
 )
 
+swift_rules_dependencies()
```

Comment:
Why is this here? Is this prepping the dependencies for the swift target above (`build_bazel_rules_swift`)?  I'm afraid I don't understand how all these functions and things work together. Maybe some inline comments?

Reply:
Re: this and the comment below.

https://github.com/bazelbuild/rules_apple#quick-setup describes the setup steps for bazel's new separation of the bazel rules from the swift rules and how to configure their dependencies..

Reply:
Ok, thanks for the information. Hopefully they will host that guide on bazel.build in the future. :)